.github/workflows/create-quantic-package.yml

```diff
@@ -2,7 +2,7 @@ name: Create Quantic Package
 
 env:
   SFDX_AUTH_JWT_USERNAME: sfdc.integration.devv2.hub@coveo.com
-  SFDX_AUTH_JWT_KEY: ${{ secrets.SFDX_AUTH_PACKAGE_JWT_KEY }}
+  SFDX_AUTH_JWT_KEY: server.key
```

Comment:
If only one step (or combinable steps) requires those environment variables, then I suggest moving them to that step (similarly to `e2e-quantic/action.yml`).

Reply:
Yep good tip. thanks